Projects/Playground/Playground.Forms.UI/Playground.Forms.UI.csproj

```diff
@@ -18,6 +18,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Update="GlobalStyles.xaml">
```

Comment:
This is the default action for XAML pages. I think we can remove these inclusions in the csproj file.